.github/workflows/dockercompose-validation.yml

```diff
@@ -16,7 +16,3 @@ jobs:
       - name: Verify development docker-compose
```

Comment:
Given that there's only one `docker-compose.yml` now, then perhaps remove the reference to `development` from here as well?